nilearn/decoding/tests/test_surf_searchlight.py

```diff
@@ -13,7 +14,7 @@
 
 
 from nilearn.surface.tests.test_surface import _generate_surf
-from nilearn.decoding.surf_searchlight import SurfSearchLight
+from nilearn.decoding.surf_searchlight import SurfSearchLight, _apply_surfmask_and_get_affinity
```

Comment:
PEP8 standards dictate that each line has no more than 80chars (col 79).
You should rewrite this, and others such as this, as
```python
from nilearn.decoding.surf_searchlight import (SurfSearchLight, 
                                            _apply_surfmask_and_get_affinity,)
```